crates/sb_core/js/user_runtime_loader.js

```diff
@@ -0,0 +1,9 @@
+// This file is meant to only have `userRuntimeCleanUp`
+// The code should address any user specific runtime behavior
+// As well as deletions
+
+function loadUserRuntime() {
+    delete globalThis.EdgeRuntime;
```

Comment:
@inian @laktek Everything that's inside this function will be called when the "User runtime" is created. For example, this cleans up the existence of `EdgeRuntime` in the user worker while using the same snapshot.

Reply:
Next step here is unifying the initialization code. Will be done in #26